openedx/core/djangoapps/verified_track_content/partition_scheme.py

```diff
@@ -14,7 +14,6 @@
     get_masquerading_user_group,
```

Comment:
@dyudyunov why leave the partitions schemes in place?  Are they still being used somewhere that we need to maintain for some reason?

Reply:
They are used for EnrollmentTrack functionality. I thought the enrollment track was introduced to replace verified track cohorts and shouldn't be deleted. Am I wrong here?

Reply:
I'm trying to figure that out now, if we're not auto assigining partitions than having the partition scheme remain may not be useful. We can certainly deal with that as a follow-up but making a note here so that we don't forget about it.